src/config/urls.ts

```diff
@@ -2,8 +2,14 @@ const BASE_URL = "https://api.escuelajs.co/api/v1";
 
 const createUrl = (url: string) => `${BASE_URL}/${url}`;
 
+type productParamsType = {
+  offset: number;
+  limit: number;
+  title: string;
+};
+
 export const urls = {
-  products: (offset: number, limit = 20) =>
-    createUrl(`products?offset=${offset}&limit=${limit}`),
+  products: ({ offset, limit, title }: productParamsType) =>
```

Comment:
лучше оставить логику формирования query-строки оставить на axios, то есть тут оставить
```suggestion
  products: () => createUrl('products')
```
А в axios прокинуть параметры как объект и он из них сам сделать строку